src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_behaviorinterface.py

```diff
@@ -89,14 +89,17 @@ def get_metadata_schema(self) -> dict:
         }
         return metadata_schema
 
-    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict, normalize_timestamps: bool = False):
```

Comment:
Yeah... but in this case I think it would really be awkward. There's lots of different timestamps that need to be normalized to the start time, I'd have to open up the file and read it twice, and I'm not even aligning the timestamps to a different stream, just normalizing them to the appropriate start time.